### DISCUSSION TITLE
i20240310 : official RPI 7" touchscreen no longer works

### DISCUSSION THREAD

**Discussion by Garrido33** [rern/rAudio#1626] [Fixed Issues]
Version i20240310 and its latest update work perfectly on my computer and smartphone, but not on my official RPI 7" touchscreen, which remains totally black. I've tried an older version to check that it's not the screen that's the problem : it works perfectly.

**Comment by R101G** [comment]
Just for information, I have updated my 7" screen installation to 20240312 and rebooted. It seems to work OK on a Pi4.

**Comment by ceejayemm** [comment]
Official 7" touchscreen with RPi 3B+ works just fine for me with 20240310 / 20240312

Regards.

Chris

**Reply by Garrido33** [reply]
Thanks R101G and ceejayemm. 
That's strange, my RPi is the 3B+ too. I'll look further and flash the micro SD card again with 20240310 / 20240312.

**Reply by R101G** [reply]
@Garrido33 It might be that the browser has been turned off in the settings. Have you checked this from a remote browser?

**Reply by Garrido33** [reply]
@R101G  The browser is not turned off. Browsing works perfectly (and is very fast) with a remote browser. I will try to reflash first.

**Reply by ceejayemm** [reply]
As @R101G says check that Features / 'Browser on RPi' is actually enabled to allow the local browser on your RPi / Touchscreen to work.  It isn't unknown for some of these features to mysteriously turn themselves on/off with updates to rAudio :-)  

Browsing on your PC/ Smartphone does not need this setting enabled to work.

Chris

**Reply by Garrido33** [reply]
Thanks. The 20240108 version with update 20240312 works perfectly on my touchscreen. The latest version 20240310 with update still doesn't work on my screen despite a total reflash of the card. Browser on RPi is enabled and all the parameters adjusted. I don't understand.

**Comment by Mightyflea64** [comment]
Same problem here using my RPi 2b with the official RPi 7" touchscreen. I tried a fresh install of the versions 20230918, 20231022, 20231128 and 20240108. All these versions work fine, but cannot be updated anymore. A fresh install of version 20240310 leaves me with a black screen, also after updating it to the latest version 20240316 via the PC web interface.
Disabling/enabling the 'Browser on RPi' has no effect.

**Comment by rern** [comment]
**rAudio** - `i20240310` 
- Browser on RPi  (Tested via HDMI)
    - `rAudio-64bit-20240310.img.xz` on RPi 4B - Starts successfully
    - `rAudio-RPi2-20240310.img.xz` on RPi 3B+ - Starts successfully
- `pacman -Syu`
    - 64bit - Restarts successfully
    - RPi2 - Restart failed - Run `pacman -Syu --ignore libunwind` instead

Downgrade for already upgraded RPi2:
```sh
wget https://alaa.ad24.cz/packages/l/libunwind/libunwind-1.7.2-1-armv7h.pkg.tar.xz
pacman -U *.xz
```

**Reply by Garrido33** [reply]
@rern

Tried rAudio-RPi2-20240310.img.xz on RPi 3B. 
Official 7" touchscreen remain black.

**Reply by emesimon** [reply]
The Official 7" touchscreen with RPI 3B+ don’t work for me with 20240310 / 20240312

**Reply by emesimon** [reply]
The touchscreen remain black.

**Reply by rern** [reply]
Please try with the image that works:
- `pacman -Sy linux-rpi` > reboot
- `pacman -S linux-firmware linux-firmware-whence` > reboot
- `pacman -S firmware-raspberrypi` > reboot
- `pacman -S raspberrypi-bootloader` > reboot

If it's still good, it's not kernel/firmware related. Then Firefox itself: `pacman -S firefox` > reboot

**Reply by Mightyflea64** [reply]
I checked the suggested updates with the following result:

- I started with a fresh install of version 20240108 on my RPi 2b with the official RPi 7" touch-screen.
- pacman -Sy linux-rpi > reboot --> Result OK
- pacman -S linux-firmware linux-firmware-whence > reboot --> Result OK
- pacman -S firmware-raspberrypi > reboot --> Result OK
- pacman -S raspberrypi-bootloader > reboot --> Result NOT OK; the screen stays black on reboot, webinterface is OK

So the problem occurs when updating the bootloader.
Any idea for a solution also for the latest version clean installation?

Thank you rern!

**Reply by Mightyflea64** [reply]
I forgot to mention; I have a RPi 2b with the BCM2836.
